src/System.Reflection.Extensions/src/System.Reflection.Extensions.builds

```diff
@@ -9,6 +9,10 @@
     <Project Include="System.Reflection.Extensions.csproj">
       <TargetGroup>netcore50</TargetGroup>
```

Comment:
We should be able to.  I was working on this one separately before I realized there was an open PR and found that the CoreCLR implementations were identical.  That'd eliminate the need for the separate project.json as well.